test/system/helpers.network.pasta.bash

```diff
@@ -0,0 +1,16 @@
+# -*- bash -*-
```

Comment:
I agree with @edsantiago here this change should be moved to the existing helpers.network instead of creating another file.